From linux-usb-devel-bounces@lists.sourceforge.net Sun Aug 20 07:59:06 2006
Date: Sun, 20 Aug 2006 18:58:42 +0400
From: Alexey Dobriyan <adobriyan@gmail.com>
To: Andrew Morton <akpm@osdl.org>
Message-ID: <20060820145842.GB5079@martell.zuzino.mipt.ru>
Content-Disposition: inline
Cc: linux-usb-devel@lists.sourceforge.net
Subject: USB: Turn usb_resume_both() into static inline

drivers/usb/core/hub.c: In function `hub_events':
drivers/usb/core/hub.c:2591: warning: statement with no effect

Signed-off-by: Alexey Dobriyan <adobriyan@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/usb.h |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/core/usb.h
+++ gregkh-2.6/drivers/usb/core/usb.h
@@ -40,7 +40,10 @@ extern int usb_port_resume(struct usb_de
 #else
 
 #define usb_suspend_both(udev, msg)	0
-#define usb_resume_both(udev)		0
+static inline int usb_resume_both(struct usb_device *udev)
+{
+	return 0;
+}
 #define usb_port_suspend(dev)		0
 #define usb_port_resume(dev)		0
 
